contracts/modules/capital/MCR.sol

```diff
@@ -199,53 +140,43 @@ contract MCR is Iupgradable {
 
   }
 
-  /**
-   * @dev Adds MCR Data. Checks if MCR is within valid
-   * thresholds in order to rule out any incorrect calculations
-   */
-  function _addMCRData(
-    uint len,
-    uint64 newMCRDate,
-    bytes4[] memory curr,
-    uint mcrE,
-    uint mcrP,
-    uint vF,
-    uint[] memory _threeDayAvg
-  )
-  internal
-  {
-    uint lowerThreshold = 0;
-    uint upperThreshold = 0;
-
-    if (len > 1) {
-      uint vtp = pool.getPoolValueInEth();
-      (lowerThreshold, upperThreshold) = getThresholdValues(vtp, vF, getAllSumAssurance(), pd.minCap());
-    }
+  function updateMCR(uint poolValueInEth) external onlyInternal {
```

Comment:
untested, want to validate the approach first, will iron out bugs later

this means:
MCR stays static until updated. sell and buy update it at MCR intervals of at least 1 hour (minUpdateTime). between 

Other alternative explored:

make `getMCR` compute the 'view' value in Pool.sol 
getMCR computes the "real" value of the MCR at all times and commits it once in a while to storage (if not written to storage lastTimeUpdated doesn't get changed).

the computation of that value is expensive and if it is incurred on every buy/sell NXM adds 40-50k gas  with this current implementation (some optimizations may be possible).

The added benefit of this is not significant, except that the MCR slowly changes and looks more "smooth" instead of getting bumped up/down sometimes

Reply:
what's the max bump?